web/web.go

```diff
@@ -111,6 +112,8 @@ func NewWebService(o *WebServiceOptions) *WebService {
 		router.Post("/-/quit", ws.quitHandler)
 	}
 
+	router.Get("/federate", instr("federate", o.Federation))
```

Comment:
Still ambivalent about this as well. What do you mean with they won't synchronize well? What would be the downside of having it in the v1 API? Higher risk of having to still change it in a breaking way later?